test/unit/credit_card_methods_test.rb

```diff
@@ -506,6 +506,38 @@ def test_should_detect_panal_card
     assert_equal 'panal', CreditCard.brand?('6020490000000000')
   end
 
+  def test_detecting_full_range_of_verve_card_numbers
+    verve = '506099000000000'
+
+    assert_equal 15, verve.length
+    assert_not_equal 'verve', CreditCard.brand?(verve)
+
+    while verve.length < 19
+      verve << '0'
+      assert_equal 'verve', CreditCard.brand?(verve), "Failed for bin #{verve}"
+    end
+
+    assert_equal 19, verve.length
+
+    verve << '0'
+    assert_not_equal 'verve', CreditCard.brand?(verve)
+  end
+
+  def test_should_detect_verve
+    assert_equal 'verve', CreditCard.brand?('5060990000000000')
```

Comment:
💬 Comment Style:

maybe an array of cc numbers + an each to check?

```ruby
  def test_should_detect_verve
    credit_cardsdit_cards = %w[5060990000000000
             506112100000000000
             5061351000000000000
             5061591000000000
             506175100000000000
             5078801000000000000
             5079381000000000
             637058100000000000
             5079400000000000000
             507879000000000000
             5061930000000000
             506136000000000000]
    credit_cards.all? { |cc| CreditCard.brand?(cc) == 'verve' }
  end

```

Reply:
Corrected! 👍